### DISCUSSION TITLE
Hackerone Community Edition

### DISCUSSION THREAD

**Discussion by k4b7** [KaTeX/KaTeX#3044] [Ideas]
It would be good to have a KaTeX specific Hackerone instead of linking to Khan Academy's Hackerone for reporting security issues.  It'll give KaTeX maintainers easy visibility of reported issues.

Here's a link to a page with info about the Hackerone Community Edition: https://www.hackerone.com/company/open-source-community

**Comment by edemaine** [comment]
Sounds good! I just applied for the KaTeX project.

**Reply by k4b7** [reply]
@edemaine have you heard back from the Hackone people regarding the application?

**Reply by edemaine** [reply]
@kevinbarabash Hmm, no. I just sent in another application to ask about the status.
